aws/networking.tf

```diff
@@ -0,0 +1,33 @@
+# Create the network to host Slurm
+resource "aws_vpc" "vpc_network" {
+  cidr_block = "10.0.0.0/16"
+  enable_dns_hostnames = "true"
+  tags = {
+    Name = "citc-net-${local.cluster_id}"
+    cluster = local.cluster_id
+  }
+}
+
+resource "aws_subnet" "vpc_subnetwork" {
+  vpc_id     = aws_vpc.vpc_network.id
+  cidr_block = "10.0.1.0/24"
```

Comment:
Is this a limit to the size of the cluster? a /24 is only 256 addresses, 254 usable.

Reply:
I think it was arbitrarily chosen. I'll increase it to a `/16` or a `/8`. I assume that `10.0.0.0/8` is possible?

Reply:
I think the subnet would need to be smaller than the containing network (currently /16)

Reply:
I've made the VPC a `10.0.0.0/8` and the subnet a `10.1.0.0/16` for now. This matches Google and Oracle.

Reply:
It seems that AWS has [a limit](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#VPC_Sizing) of `/16` for a VPC. I'll make the VPC `10.0.0.0/16` and the subnet `10.0.0.0/17` for now.